unison-cli/src/Unison/Cli/MonadUtils.hs

```diff
@@ -279,16 +316,15 @@ expectLatestTypecheckedFile :: Cli r (TypecheckedUnisonFile Symbol Ann)
 expectLatestTypecheckedFile =
   getLatestTypecheckedFile & onNothingM (Cli.returnEarly Output.NoUnisonFile)
 
--- | Update a branch at the given path, returning `True` if
+-- Update a branch at the given path, returning `True` if
 -- an update occurred and false otherwise
 updateAtM ::
   Text ->
   Path.Absolute ->
   (Branch IO -> Cli r (Branch IO)) ->
   Cli r Bool
 updateAtM reason (Path.Absolute p) f = do
-  loopState <- get
-  let b = loopState ^. #lastSavedRoot
+  b <- getRootBranch
```

Comment:
Is this behavior change significant in some way? Before we got the last *saved* root, now we get the last in-memory root.

Reply:
I noticed that when writing, TBH I. think the new behaviour is preferred,  since previously any in-memory updates would just be thrown away, which seems incorrect. But it doesn't seem to have changed any transcripts at any rate.

It is indeed different though.